tests/cli/clean/conftest.py

```diff
@@ -1,14 +1,14 @@
 """Fixtures for cli clean tests"""
 
 import pytest
+from tests.cli.workflow.balsamic.conftest import *
```

Comment:
should we not just import what we need?

Reply:
I will rethink this. Currently I wanted to import balsamic context, but it had many other dependencies it needed in conftest. It might be just easier to build a separate fixture for the clean command